WebSharper.UI.CSharp.Templating.ServerSide.Tests/WebSharper.UI.CSharp.Templating.ServerSide.Tests.csproj

```diff
@@ -22,6 +22,11 @@
     <ProjectReference Include="../WebSharper.UI.CSharp\WebSharper.UI.CSharp.fsproj" />
     <ProjectReference Include="../WebSharper.UI.Templating.Runtime\WebSharper.UI.Templating.Runtime.fsproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@diegopego These are the project file changes that are interfering and should be reverted, thanks. `paket.lock` update does the trick on its own.

Reply:
Thanks. I'm sorry. I'm very new to fsharp. I had never used paket before. I made a new PR reverting the improper references and added the packet dependency.

Reply:
Sure!

Reply:
@granicz , I said PR, when in reality, I have made two new commits on this PR.